WORKFLOW.md

```diff
@@ -130,13 +122,29 @@ git commit -m "fix the problem"
 git push
 ```
 
+Another use case is when a past release needs a patch. For example, we are currently on version 3.2 but find a critical 
+bug that is present since 2.5 and want to fix it. Then we would create a hotfix branch and release it as 2.5.1.
+
+### Working locally
+
+```
+# create a hotfix branch based on master, because master is what will be deployed to production
+git checkout master@2.5
+git fetch
+git pull origin master
+git checkout -b hotfix/describe-the-problem
+
+git add patch.fix
+git commit -m "fix the problem"
+git push
+```
+
 Don't forget to update the Changelog and the version in `setup.py`.
 
 ### Github workflow
 
 - Open a Pull Request against `master`
 - When the PR's approved and the code is tested, `squash and merge` to squash your commits into a single commit.
-- Open a Pull Request against `staging` (may need to rebase first).
 - A tag will automatically be triggered in our CI/CD. This tag/release will use the version for its title and push a new version
 of Butterfree's python package to our private server.
```

Comment:
I think we can't do this when fixing old versions. Since we can't merge old things in current master.